### PR TITLE
refactor(telecom.telephony): extract tab links through the routing

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.controller.js [hunk 1]
```diff
@@ -2,85 +2,116 @@ import filter from 'lodash/filter';
...
+    $q,
+    $translate,
+    $stateParams,
+    abbreviatedNumbersLink,
+    administrationLink,
+    billingLink,
+    billingAccountLink,
+    currentActiveLink,
+    isBetaActive,
+    guidesLink,
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.controller.js]: Shouldn't be placed right before `isBetaActive`?

File: packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html [hunk 2]
```diff
@@ -23,44 +23,53 @@
                 <oui-header-tabs>
                     <oui-header-tabs-item
                         data-text="{{ 'telephony_group_tab_dashboard' | translate }}"
-                        data-state="telecom.telephony.billingAccount">
+                        data-href="{{:: BillingAccountCtrl.billingAccountLink }}"
+                        data-active = "BillingAccountCtrl.currentActiveLink() === BillingAccountCtrl.billingAccountLink">
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html]: :warning: Caution there are some extra spaces.

```suggestion
                        data-active="BillingAccountCtrl.currentActiveLink() === BillingAccountCtrl.billingAccountLink">
```

Comment (FredericEspiau) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html]: Where is the Prettier poc ? @antleblanc  😅

Comment (marie-j) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html]: This should have been detected by a linter 🤔

Comment (antleblanc) [reply packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html]: It's a valid syntax!
But to keep consistency please prefer using the same coding style.

**Pull request merged**